tsdb/head.go

```diff
@@ -209,6 +213,20 @@ func newHeadMetrics(h *Head, r prometheus.Registerer) *headMetrics {
 		Name: "prometheus_tsdb_checkpoint_creations_total",
 		Help: "Total number of checkpoint creations attempted.",
 	})
+	m.lowWatermark = prometheus.NewGaugeFunc(prometheus.GaugeOpts{
```

Comment:
Silly question, but do we need to keep those variables? We could just `MustRegister(prometheus.NewGaugeFunc...., ...)`, right?

Reply:
That's true. The functional metrics never need to be referenced. I'll register them directly and remove them from the struct.

Reply:
Unless we need them in tests (which is the case for `m.series`).